requirements/requirements.txt

```diff
@@ -201,10 +201,8 @@ grpcio==1.62.2
     # via
     #   -r /awx_devel/requirements/requirements.in
     #   opentelemetry-exporter-otlp-proto-grpc
-hiredis==2.0.0
-    # via
-    #   -r /awx_devel/requirements/requirements.in
-    #   aioredis
+hiredis==2.3.2
+    # via aioredis
```

Comment:
since aioredis is removed from the channel_redis upgrade to 4.x we can probably just remove hiredis

Reply:
updator need to be rerun after the channel_redis upgrade is merged

Reply:
@jessicamack can u roll this into the channel_redis PR?

Reply:
yes, we'll have to merge them in order but we can leave this one for last and run update so the requirements.txt is up to date